Osiris/Hacks/Backtrack.cpp

```diff
@@ -108,8 +112,53 @@ void Backtrack::run(UserCmd* cmd) noexcept
     if (bestRecord) {
         auto record = records[bestTargetIndex][bestRecord];
         memory->setAbsOrigin(bestTarget, record.origin);
-        cmd->tickCount = timeToTicks(record.simulationTime + getLerp());
+        cmd->tickCount = timeToTicks(record.simulationTime);
+    }
+}
+
+void Backtrack::AddLatencyToNetwork(NetworkChannel* network, float latency) noexcept
+{
+    for (auto& sequence : sequences)
+    {
+        if (memory->globalVars->serverTime() - sequence.servertime >= latency)
+        {
+            network->InReliableState = sequence.inreliablestate;
+            network->InSequenceNr = sequence.sequencenr;
+            break;
+        }
+    }
+}
+
+void Backtrack::UpdateIncomingSequences(bool reset) noexcept
+{
+    static float lastIncomingSequenceNumber = 0.f;
+
+    if (reset)
+        lastIncomingSequenceNumber = 0.f;
+    int timeLimit = config->backtrack.timeLimit; if (timeLimit <= 200) { timeLimit = 0; } else { timeLimit - 200; }
+    if (!config->backtrack.fakeLatency || timeLimit == 0)
+        return;
+
+    if (!localPlayer)
+        return;
+
+    auto network = interfaces->engine->getNetworkChannel();
+    if (!network)
+        return;
+
+    if (network->InSequenceNr > lastIncomingSequenceNumber)
```

Comment:
Check if it isnt equal, works better

Reply:
solution = 
 if (network->InSequenceNr != lastIncomingSequenceNumber){
?

Reply:
yeah